src/prefect/cli/server.py

```diff
@@ -80,13 +80,27 @@ def generate_welcome_blurb(base_url, ui_enabled: bool):
         """
     )
 
+    from prefect.server.utilities.database import get_dialect
+    from prefect.settings import PREFECT_API_DATABASE_CONNECTION_URL
+
+    using_sqlite = (
+        get_dialect(PREFECT_API_DATABASE_CONNECTION_URL.value()).name == "sqlite"
```

Comment:
might this `PREFECT_API_DATABASE_CONNECTION_URL` value sometimes be `None`? i noticed the setting is `Optional`